spec/models/abuse_report_spec.rb

```diff
@@ -155,8 +155,8 @@
       it_behaves_like "alright", "http://archiveofourown.org/users/someone"
 
       context "a month later" do
-        before { Timecop.freeze(32.days.from_now) }
-        after { Timecop.return }
+        before { travel(32.days) }
+        after { travel_back }
```

Comment:
Time stubs are automatically removed at the end of the test, so you can skip the `after { travel_back }`s.